upnp/src/genlib/net/http/webserver.c

```diff
@@ -1367,6 +1367,15 @@ static int process_request(
 		/*          goto error_handler; */
 		/*      } */
 	}
+	{
+		UpnpListHead *eHeader =
+			UpnpFileInfo_get_ExtraHeadersList(finfo);
+		UpnpExtraHeaders *h = UpnpExtraHeaders_new();
+		UpnpExtraHeaders_set_resp(h, "Access-Control-Allow-Origin: *");
```

Comment:
Is this allowing *any server anywhere on the web* to access the UPnP resources? I that's the case  I don't believe that's desirable.

(I am not sure what is supposed to happen when the other `Access-Control-Allow-*` are absent).

Reply:
It's mainly controlling the access of scripts from other sources, a.k.a side-loading

Reply:
Won't this make it easier for a remote web server to scan your LAN (or local machine):

- for fingerprinting your LAN/device (eg. when you are using a VPN),
- for detecting the IP address, port and version of UPnP software in order to try to trigger an attack?